frontend/src/static/js/components/webstatus-overview-cells.ts

```diff
@@ -117,25 +141,45 @@ const renderFeatureName: CellRenderer = (feature, routerLocation, _options) => {
 
 const renderBaselineStatus: CellRenderer = (
```

Comment:
Since there are a lot of different combinations that can be rendered, we should have some coverage for this.

Feel free to either:
1. add a unit test to the existing cells test [file](https://github.com/GoogleChrome/webstatus.dev/blob/main/frontend/src/static/js/components/test/webstatus-overview-cells.test.ts) that asserts the different combinations (using the cases I mentioned in the other comment).
2. Add a playwright test that asserts the different combinations (using the cases I mentioned in the other comment)